tests/Controller/CRUDControllerTest.php

```diff
@@ -3644,6 +3644,114 @@ public function testBatchActionWithConfirmation(): void
         $this->assertSame('@SonataAdmin/CRUD/batch_confirmation.html.twig', $this->template);
     }
 
+    /**
+     * NEXT_MAJOR: Remove this legacy group.
```

Comment:
I'm not sure why the legacy group is here. Is there any instruction that will be resolved in the next major about these changes?

Reply:
These two tests are copy/paste of `CRUDControllerTest::testBatchActionWithConfirmation()`.